nobuco/node_converters/tensor_getattr.py

```diff
@@ -0,0 +1,21 @@
+from typing import Optional
+
+import torch
+from torch import Tensor
+
+import tensorflow as tf
+
+from nobuco.commons import ChannelOrderingStrategy
+from nobuco.converters.node_converter import converter
+
+
+@converter(torch.Tensor.__getattribute__, channel_ordering_strategy=ChannelOrderingStrategy.FORCE_PYTORCH_ORDER)
+def converter_getattr(self: Tensor, name: str, *, out: Optional[Tensor] = None):
+    def func(self, name: str, *, out: Optional[Tensor] = None):
+        if name == "real":
```

Comment:
You'll have to copy `T` and `data` attributes from the original `converter_getattribute`, and then remove it so there is only one converter.

Reply:
``` python
# script
real = torch.tensor([1, 2], dtype=torch.float32)
imag = torch.tensor([3, 4], dtype=torch.float32)
complex_tensor = torch.complex(real, imag)
tf_real = tf.math.__getattribute__('real')(complex_tensor)
tf_imag = tf.math.__getattribute__('imag')(complex_tensor)
tf_real, tf_imag
```
```
# output
(<tf.Tensor: shape=(2,), dtype=float32, numpy=array([1., 2.], dtype=float32)>,
 <tf.Tensor: shape=(2,), dtype=float32, numpy=array([3., 4.], dtype=float32)>)
```

```
# converter
@nobuco.converter(torch.Tensor.__getattribute__, channel_ordering_strategy=ChannelOrderingStrategy.MINIMUM_TRANSPOSITIONS)
def converter_getattr(self: torch.Tensor, name: str):
    return tf.math.__getattribute__(name)(self)
```

I thought the converter should work theoretically, but it doesn't with the error
```
TypeError: 'tensorflow.python.framework.ops.EagerTensor' object is not callable
```

For quick reproduction, visit the colab [Notebook](https://colab.research.google.com/drive/1ni76Z6eY4flLA4cSoIBZkjQzWOMlaF_1#scrollTo=gromDhTIkIV2).

It seems to require debugging. If you don't mind, I'll debug it when I have time, or you can handle it for the agile closure.